rotkehlchen/api/v1/schemas.py

```diff
@@ -1951,3 +1951,7 @@ def validate_schema(  # pylint: disable=no-self-use
 
 class AssetsImportingFromFormSchema(Schema):
     file = FileField(allowed_extensions=['.zip', '.json'], required=True)
+
+
+class DBSnapshotQuerySchema(Schema):
```

Comment:
This is not wrong, but we can just simplify the code a bit.

There is already a schema with the exact same schema (just a single timestamp) here: https://github.com/rotki/rotki/blob/963c13219c308f1efd19724e6793a1013482ddc5/rotkehlchen/api/v1/schemas.py#L1764-L1765

How about renaming it from `TimedManualPriceSchema` to something like `SingleTimestampSchema` and then use that instead in the resources you made and wherever `TimedManualPriceSchema` was used?